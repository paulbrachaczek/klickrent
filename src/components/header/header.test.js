import {shallow} from 'enzyme';
import toJson from "enzyme-to-json";
import React from 'react';
import Header from "./header.component.jsx";

it("renders correctly", () => {
    const HeaderComponent = shallow(<Header />);
    expect(toJson(HeaderComponent)).toMatchSnapshot();
});
  