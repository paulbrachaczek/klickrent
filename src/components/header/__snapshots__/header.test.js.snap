// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header
  className="o-header"
>
  <h1>
    klickrent
  </h1>
</header>
`;
